### PR TITLE
WIP Add commit validation for special words

### PR DESCRIPTION
**Pull request by mdellweg**
TODO: Make the corresponding WF job mandatory.

### CODE REVIEW

File: .ci/scripts/validate_commit_message.py [hunk 1]
```diff
@@ -7,6 +7,12 @@
 from github import Github
 
 KEYWORDS = ["fixes", "closes"]
+BLOCKING_REGEX = [
+    "WIP",
+    "NOMERGE",
+    r"DO\s*NOT\s*MERGE",
+    "EXPERIMENT",
+]
```
Comment (mdellweg) [inline .ci/scripts/validate_commit_message.py:15]: Please comment with your favourites.

**Comment by mdellweg** [comment]
I intend to use this PR as a guinea pig for auto-merge. Please do not merge manually.

**Review by ggainey — APPROVED** [review]
lg2m - let's see how it goes

**Review by ggainey — DISMISSED** [review]
lg2m

**Review by ggainey — APPROVED** [review]
lg2m again

**Pull request merged**